plugins/inventory/nb_inventory.py

```diff
@@ -445,26 +456,36 @@ def group_extractors(self):
         # Locations were added in 2.11 replacing rack-groups.
         if self.api_version >= version.parse("2.11"):
             extractors.update(
-                {"location": self.extract_location,}
+                {
+                    "location": self.extract_location,
+                }
```

Comment:
This change appears to be unrelated to your patch series, and should not have caused an issue with `black` - please remove this change and the other changes like this that are unrelated.

Reply:
I didn't notice that black had modified anything outside of the lines I changed / added.
I will get this changed back to its original formatting.